#! /bin/sh /usr/share/dpatch/dpatch-run
## src_Makefile.dpatch by Mehdi Dogguy <dogguy@pps.jussieu.fr>
##
## DP: Do not try to build .cmx on non-native architectures.

@DPATCH@
diff -urNad menhir~/src/Makefile menhir/src/Makefile
--- menhir~/src/Makefile	2009-01-19 14:24:29.000000000 +0100
+++ menhir/src/Makefile	2009-01-23 11:21:11.000000000 +0100
@@ -246,5 +246,9 @@
 
 # Building menhirLib.
 
+ifeq ($(TARGET),byte)
+lib: menhirLib.cmo menhirLib.cmi
+else
 lib: menhirLib.cmo menhirLib.cmi menhirLib.cmx menhirLib.o
+endif
 
